subprojects/core/src/main/java/org/gradle/api/internal/project/taskfactory/AbstractOutputPropertyAnnotationHandler.java

```diff
@@ -40,7 +38,6 @@ public void validate(String propertyName, Object value, Collection<String> messa
             public void update(TaskInternal task, final Callable<Object> futureValue) {
                 createPropertyBuilder(context, task, futureValue)
                     .withPropertyName(context.getName())
-                    .withPathSensitivity(getPathSensitivity(context))
```

Comment:
Doesn't look like we ever expose `TaskFilePropertyBuilder` without someone explicitly asking for it. So yeah, let's remove the method.